test/guest/fedora-atomic.setup

```diff
@@ -18,12 +18,17 @@
 # You should have received a copy of the GNU Lesser General Public License
 # along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
 
+# Use mac's in name policy, prevents interference from network-scripts
```

Comment:
However, we will never receive any metrics for the interfaces, so wait_for_iface will fail.  Do we expect metrics to work on Atomic?  I'll look into this.